vpr/src/place/timing/delay_model/PlacementDelayModelCreator.h

```diff
@@ -0,0 +1,31 @@
+
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include "netlist.h"
+
+class PlaceDelayModel;
+struct t_placer_opts;
+struct t_router_opts;
+struct t_det_routing_arch;
+struct t_segment_inf;
+struct t_chan_width_dist;
+struct t_direct_inf;
+
+class PlacementDelayModelCreator {
+  public:
+    // nothing to do in the constructor and destructor
+    PlacementDelayModelCreator() = default;
```

Comment:
I think you mean to delete the constructor? Since this class only contains a static method, do you expect someone to create an object of this class? If not, deleting the constructor makes this explicit.